pkgs/applications/virtualization/virt-viewer/default.nix

```diff
@@ -54,6 +60,10 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  mesonFlags = [
```

Comment:
By convention, flags go after inputs, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.

Reply:
Could we get that documented in a more official place? I've never seen this.

IMO configure flags should go before inputs because I like to put attrs roughly in the order they're used. In my view, flags are used first to pass onto the configure tool (nativeBuildInputs) which in-turn then uses buildInputs in the buildPhase.

I'd love to adhere to a convention/standard here but only if it's decided by a larger sample of the community and documented properly.

Reply:
It is only an empirical observation of the most common of several attribute orderings used in Nixpkgs. But setting it formally would require an RFC, for which we lack bandwidth at the moment. So until that is done, us GNOME maintainers decided to enforce it in GNOME-adjacent packages.

Reply:
If the vast majority of GNOME maintainers agree on a style for GNOME-adjacent packages only, that's fine by me. I don't think that needs an RFC. In my eyes, an RFC would only be necessary if you wanted to decide this e.g. for all of Nixpkgs, not just a pretty homogenic subset.

Reply:
I just want to have it documented more officially than a discourse post that looks like a question rather than a standard. Just a simple `pkgs/desktop/gnome/README.md` would be fine by me.